src/main/java/net/imglib2/algorithm/hough/HoughTransforms.java

```diff
@@ -136,6 +138,7 @@ public static long[] getVotespaceSize( final Dimensions dimensions, final int nT
 	 * Returns the size of the voteSpace output image given desired {@code nRho}
 	 * and {@code nTheta} values.
 	 *
+	 * @implNote op names='features.hough.getVotespaceSize', type=Function
```

Comment:
The `features.hough.getVotespaceSize` is not idea. At a minimum, change it to `features.hough.linearVotespaceSize`, but I feel like there is a better classification here. Thoughts, anyone?

Reply:
Look to the method name for guidance. It's `...hough.HoughTransforms.getVotespaceSize`. The `get` is just a Javaism, and we need the `features.` Op prefix for organization, so that leaves `features.hough.votespaceSize`.

In other words: if this is only the *linear* votespace size calculation, and there are also *non-linear* ones, it is not only the name of the Op that will need to change, but also the static method name itself here, no?

Reply:
For now, we have decided not to opify hough transforms.

Reply:
We did? Maybe I misunderstood the scope of these methods, then...